core-primitives/enclave-api/src/remote_attestation.rs

```diff
@@ -409,27 +409,27 @@ impl RemoteAttestation for Enclave {
 				 pck_crl_size: {}\n, \
 				 tcb_info_issuer_chain: {:?}\n, \
 				 tcb_info_issuer_chain_size: {}\n, \
-				 tcb_info: {:?}\n, \
+				 tcb_info: {}\n, \
 				 tcb_info_size: {}\n, \
 				 qe_identity_issuer_chain: {:?}\n, \
 				 qe_identity_issuer_chain_size: {}\n, \
-				 qe_identity: {:?}\n, \
+				 qe_identity: {}\n, \
 				 qe_identity_size: {}\n",
 				collateral.version,
 				collateral.tee_type,
-				std::ffi::CStr::from_ptr(collateral.pck_crl_issuer_chain).to_str().unwrap(),
+				std::ffi::CStr::from_ptr(collateral.pck_crl_issuer_chain).to_string_lossy(),
```

Comment:
I think this will need a tad bit more robust solution (e.g. handling invalid values without new allocations as they will crash the enclave), but LGTM otherwise!